src/ic-websocket-gateway/src/gateway_metrics.rs

```diff
@@ -8,21 +8,32 @@ use metrics_util::{MetricKindMask};
 pub fn init_metrics(port: Option<u16>) -> Result<(), Box<dyn Error>> {
     let builder = PrometheusBuilder::new()
         .with_http_listener(SocketAddr::new(IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)), port.unwrap_or(9000)));
+
+    // Set the idle timeout for counters and histograms to 30 seconds then the metrics are removed from the registry
     builder
         .idle_timeout(
             MetricKindMask::ALL,
-            Some(Duration::from_secs(10)),
+            Some(Duration::from_secs(30)),
         )
         .install()
         .expect("failed to install Prometheus recorder");
 
     describe_gauge!("clients_connected", "The number of clients currently connected");
+    describe_gauge!("active_pollers", "The number of pollers currently connected");
     describe_histogram!(
         "connection_duration",
         "The duration of the client connection"
     );
+    describe_histogram!(
+        "tls_resolution_time",
+        "The resolution time of the TLS handshake"
+    );
+    describe_histogram!(
+        "connection_opening_time",
+        "The time it takes to open a connection"
+    );
 
```

Comment:
No, because other times the clients_connected gauge is called up with a canister_id label, so according to the graph nothing change (without a specific label).